### PR TITLE
feat: provide full import statement range

### PR DESCRIPTION
**Pull request by nayeemrmn**
Renames `Import::range` to `Import::specifier_range`, adds `Import::full_range`.

Towards https://github.com/denoland/deno/issues/18884.

### CODE REVIEW

**Comment by nayeemrmn** [comment]
Don't know what caused this:
image

**Comment by dsherret** [comment]
Bug fix: https://github.com/denoland/deno/pull/25860

**Review by dsherret — APPROVED** [review]
LGTM

File: src/analyzer.rs [hunk 1]
```diff
@@ -172,6 +174,7 @@ pub struct DynamicDependencyDescriptor {
   pub argument: DynamicArgument,
   /// The range of the argument.
   pub argument_range: PositionRange,
+  pub range: PositionRange,
```
Comment (dsherret) [inline src/analyzer.rs:177]: I just realized this breaks the deserializtion of information we store on JSR. We'll need to think of a solution for this.

**Pull request merged**